jni/src/main/java/com/horizen/common/interfaces/DefaultReader.java

```diff
@@ -0,0 +1,34 @@
+package com.horizen.common.interfaces;
+
+import com.horizen.common.DBIterator;
+
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+
+public interface DefaultReader extends Reader, DefaultColumnFamily {
+    default Optional<byte[]> get(byte[] key){
+        return get(defaultCf(), key);
+    }
+    default Map<byte[], Optional<byte[]>> get(Set<byte[]> keys){
```

Comment:
The Map at the Rust side is used in the sc-storage implementation because in Rust the Map has not such limitations with byte arrays as in Java. So it is more convenient to directly put Key-Values into a Map and access them by Keys than doing additional parsing for Key-Value pairs that are put into a List.
And actually the transaction module in RocksDB wrapper crate (unlike C++ implementation of RocksDB) doesn't have methods for batched updating a transaction with a List of Key-Values - just a methods for updating with a single KV pair (https://github.com/HorizenOfficial/rust-rocksdb/blob/transactions/src/transactions/ops/put.rs).

And regarding the 'update' method - the Map type for KVs to insert was chosen due to Rust JNI supports parsing the Java Map objects as a sequence of KV entries, that is more convenient than additionally to parse each element of a List as a KV-pair object.

But now I see that due to the issues with 'byte[] equals' in Java it is inconvenient to use the Map type, so I can change Map and Set to Lists on Java and Rust JNI sides whenever it is needed. But there is no need to change types from Map to List in sc-storage internals, because Rust properly supports Maps with byte arrays.

Regarding the empty byte[] in returned list of values - then, I think, it should be just null, because an empty byte array seems to be one of possible values for a key. And if the returned List should contain just values, than it will be a documented assumption that the order of values is the same as the order of requested keys in parameters.